_pages/dev/seo/structured-data.md

```diff
@@ -21,50 +21,43 @@ Structured Data provides a data structure for the web that makes the web content
 
-Structured Data is optional for your project. You could even consider a combination of a tag management solution and the structured data generated in Spartacus.
+For more information, see [Tag Management System]({{ site.baseurl }}{% link _pages/dev/tag-management-system.md %}).
 
-## Schema types
+## Schema Types
 
-There are various schemas for the web, and schemas can be extended as well. The schemas are maintained by the schema.org community. THe following are common schemas for commerce:
+There are various schemas for the web, and schemas can be extended as well. The schemas are maintained by the Schema.org community. The following are common schemas for commerce: WebSite, Organization, BreadcrumbList, Product, Reviews, Offer, and Rating.
 
--   [WebSite](https://schema.org/WebSite)
--   [Organization](https://schema.org/Organization)
--   [BreadcrumbList](https://schema.org/BreadcrumbList)
--   [Product](https://schema.org/Product)
--   [Reviews](https://schema.org/Reviews)
--   [Offer](https://schema.org/Offer)
--   [Rating](https://schema.org/Rating)
+Spartacus has implemented standard schemas for the following:
 
-Spartacus has implemented standard schemas for BreadcrumbList, Product, Review, Rating and Offer. You can further enhance the generated schemas or introduce new schemas to the application as well.
+- [BreadcrumbList](https://schema.org/BreadcrumbList)
+- [Product](https://schema.org/Product)
+- [Review](https://schema.org/Review)
+- [Offer](https://schema.org/Offer)
+- [Rating](https://schema.org/Rating)
 
-The schemas offer the flexibility to add different schemas in a script tag, or wrap multiple schemas in an overarching schema. In the standard schema build for product data, the product reviews, offers and rating are part the the product schema.
+You can further enhance the generated schemas or introduce new schemas to the application as well.
 
-An alternative or additional approach to add schemas to the website is using a tag management solution. As long as we do not provide a _data layer_ to GTM, customers will not be able to easily integrate content related data.
+The schemas allow you to add different schemas in a script tag, or to wrap multiple schemas within a single, overarching schema. In the standard schema build for product data, the product reviews, offers, and rating are included as part the the product schema.
 
 ### Structured Data Vocabularies
 
-There are three vocabularies that can be used to describe Structured Data on a web page:
+There are three vocabularies that can be used to describe structured data on a web page: RDFa, Microdata, and JSON-LD.
 
--   RDFa
--   Microdata
--   JSON-LD
+Spartacus uses JSON-LD because it is the recommended format according to Google, it is an easier format to generate, and it is decoupled from the actual UI components. The other vocabularies are tightly coupled with the UI components, which has a number of disadvantages.
```

Comment:
can we change to:

Spartacus uses JSON-LD because it is an easier format to generate and it is decoupled from the actual UI components. The other vocabularies are tightly coupled with the UI components, which has a number of disadvantages.

Reply:
👍